openprescribing/frontend/tests/commands/test_import_measures_new.py

```diff
@@ -190,15 +215,246 @@ def test_import_measures(self):
+                    0,  #  male_25_34
+                    0,  #  male_35_44
+                    0,  #  male_45_54
+                    0,  #  male_55_64
+                    0,  #  male_65_74
+                    0,  #  male_75_plus
+                    0,  #  female_5_14
+                    0,  #  female_15_24
+                    0,  #  female_25_34
+                    0,  #  female_35_44
+                    0,  #  female_45_54
+                    0,  #  female_55_64
+                    0,  #  female_65_74
+                    0,  #  female_75_plus
+                    total_list_size,  #  total_list_size
+                    0,  #  astro_pu_cost
+                    0,  #  astro_pu_items
+                    '{}',  #  star_pu
+                    practice.ccg_id,  #  pct_id
+                    practice.code,  #  practice
+                ]
+
+                practice_statistics_rows.append(row)
+                practice_statistics = practice_statistics.append(
+                    {
+                        'month': timestamp[:10],
+                        'practice_id': practice.code,
+                        'ccg_id': practice.ccg_id,
+                        'stp_id': practice.ccg.stp_id,
+                        'regional_team_id': practice.ccg.regional_team_id,
+                        'total_list_size': total_list_size,
+                    },
+                    ignore_index=True,
+                )
+
+        # Upload practice_statistics_rows to BigQuery.
+        table = Client('hscic').get_or_create_table(
+            'practice_statistics', schemas.PRACTICE_STATISTICS_SCHEMA
+        )
+
+        with tempfile.NamedTemporaryFile() as f:
+            writer = csv.writer(f)
+            for row in practice_statistics_rows:
+                writer.writerow(row)
+            f.seek(0)
+            table.insert_rows_from_csv(f.name)
+
+        # Do the work.
+        call_command('import_measures', measure='coproxamol')
+
+        # Check calculations by redoing calculations with Pandas, and asserting
+        # that results match.
+        month = '2018-08-01'
+        prescriptions = prescriptions[prescriptions['month'] == month]
+        numerators = prescriptions[prescriptions['bnf_code'] == '0407010Q0AAAAAA']
+        denominators = practice_statistics[practice_statistics['month'] == month]
+        mg = MeasureGlobal.objects.get(month=month)
+
+        practices = self.calculate_practice_statistics_measure(
+            numerators,
+            denominators,
+            'practice',
+            Practice.objects.values_list('code', flat=True),
+        )
+        self.validate_practice_statistics_measure_global(mg, practices, 'practice')
+        mvs = MeasureValue.objects.filter(
+            month=month,
+            practice_id__isnull=False,
+            pct_id__isnull=False,
+            stp_id__isnull=False,
+            regional_team_id__isnull=False,
+        )
+        self.assertEqual(mvs.count(), 625)
```

Comment:
What does this number relate to - number of practices presumably? Could it be a variable? (e.g. `Practice.objects.count()`)

Reply:
Good point!